src/util/vscodePrompts.ts

```diff
@@ -11,31 +11,43 @@ export async function askForGroupID(): Promise<string | undefined> {
     value: "com.example",
     ignoreFocusOut: true,
     validateInput: (value: string) => {
```

Comment:
I think this can just become `validateInput: validateGroupId `. Same w/ artifact id